src/itemset.js

```diff
@@ -181,7 +213,7 @@ define([
             deserialize: function (data, key, mug) {
                 _.each(data[key], function (value, i) {
                     var children = mug.form.getChildren(mug),
-                        itemset = children[i] || afterDynamicSelectInsert(mug.form, mug),
+                        itemset = children[i] || afterDynamicSelectInsert(mug.form, mug, _.first(data.itemsetData)),
```

Comment:
> I'm guessing the bug is in `Itemset.spec.itemsetData.deserialize`.

Thank you - looks like there might have been an oversight in https://github.com/dimagi/Vellum/commit/e33600daf3eb3c1f417bc774b1ccbbbc20193fc9. Submitted a different fix.